app/models/agents/java_script_agent.rb

```diff
@@ -102,6 +102,7 @@ def execute_js(js_function, incoming_events = [])
           memory.to_json
         end
       end
+      context["unescapeHTML"] = lambda { |a, x| CGI.unescapeHTML(x) }
```

Comment:
Sure! escapeHTML didn't fit my use-case for this, but I can add another commit to do that.

I should also add documentation so that others can be aware of the existence of this.

Sent from my iPhone

> On May 2, 2015, at 1:10 PM, Andrew Cantino <notifications@github.com> wrote:
> 
> In app/models/agents/java_script_agent.rb:
> 
> > @@ -102,6 +102,7 @@ def execute_js(js_function, incoming_events = [])
> >            memory.to_json
> >          end
> >        end
> > +      context["unescapeHTML"] = lambda { |a, x| CGI.unescapeHTML(x) }
> Yea, it may not be the right solution. I'm fine with this addition. Do you think that we should add escapeHtml too?
> 
> —
> Reply to this email directly or view it on GitHub.
>